.circleci/config.yml

```diff
@@ -33,8 +33,8 @@ jobs:
           RAILS_ENV: test
           DATABASE_URL: "postgres://aperta@localhost:5432/aperta"
           FIREFOX_CACHE_DIR: /home/circleci/ff
-          FIREFOX_URL: https://download.mozilla.org/?product=firefox-latest&os=linux64&lang=en-US
-          GECKODRIVER_URL: https://github.com/mozilla/geckodriver/releases/download/v0.20.1/geckodriver-v0.20.1-linux64.tar.gz
+          FIREFOX_URL: https://ftp.mozilla.org/pub/firefox/releases/61.0.2/linux-x86_64/en-US/firefox-61.0.2.tar.bz2
```

Comment:
Please note in the ticket that there is nothing particular about v61.02 that caused you to lock it, so that future dev won't waste time trying to figure out why.